test/UnitTest/v1/test_casesNGSIv1.py

```diff
@@ -16,704 +16,974 @@
 print("Testing of v1 API")
 # testCase 1
 '''
-  Testing post subscription
+  To test subscription request 
 '''
 def test_getSubscription1():
-	url=brokerIp+"/ngsi10/subscribeContext"
-	headers={'Content-Type' : 'application/json'}
-	r=requests.post(url,data=json.dumps(data_ngsi10.subdata1),headers=headers)
+        url=brokerIp+"/ngsi10/subscribeContext"
+        headers={'Content-Type' : 'application/json'}
+        r=requests.post(url,data=json.dumps(data_ngsi10.subdata1),headers=headers)
         resp_content=r.content
         resInJson= resp_content.decode('utf8').replace("'", '"')
         resp=json.loads(resInJson)
-        #print(resp)
-	#print(r.status_code)
-	assert r.status_code == 200
+        print(resp)
+        assert r.status_code == 200
+
 
 #testCase 2
 '''
```

Comment:
It is not mandatory that entity should exist beforehand while creating a subscription, so the sequence of test cases in the beginning that run update, subscription and then GET subscription, is not required. Different variants of update request can be tested separately.